### PR TITLE
Updated HOP and QM cartridge and added new supply cartridge.

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do

- Removes cargo apps from the HOP's PDA cartridge (HumanResources9001)
- Added a the status display app to the QM's PDA cartridge
- Added a new supply cartridge, which is identical to the QM PDA cartridge with the exception of the status display app
- Mapped in the new cartridges

Fixes: #22432 
Fixes: #22438

## Why It's Good For The Game
The QM is now a head, and should have access to status displays like all other head. The HOP is no longer part of the supply department so they should lose the apps associated with it.

## Testing
Load in as QM.
Check my PDA.
I can change the displays.

## Changelog
:cl:
tweak: Removed cargo apps from the HOP's PDA cartridge.
tweak: Added a the status display app to the QM's PDA cartridge.
tweak: Added a new separate supply cartridge.
/:cl:

### CODE REVIEW

File: code/modules/pda/cart.dm [hunk 1]
```diff
@@ -141,13 +141,13 @@
...
-/obj/item/cartridge/quartermaster
+/obj/item/cartridge/cargo
 	name = "Space Parts & Space Vendors Cartridge"
 	desc = "A data cartridge for portable microcomputers. Has supply records and MULEbot control."
 	icon_state = "cart-q"
 	programs = list(
 		new /datum/data/pda/app/supply,
-		new /datum/data/pda/app/mule_control
+		new /datum/data/pda/app/mule_control,
 	)
```
Comment (Henri215) [inline code/modules/pda/cart.dm:151]:
```suggestion
		new /datum/data/pda/app/mule_control
	)
```

File: code/modules/pda/cart.dm [hunk 2]
```diff
@@ -158,15 +158,23 @@
 		new /datum/data/pda/app/status_display
 	)
 
+/obj/item/cartridge/qm
+	name = "Space Parts & Space Vendors Cartridge DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has supply records and MULEbot control and a status display controller."
```
Comment (Henri215) [inline code/modules/pda/cart.dm:163]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control and a status display controller."
```

**Review by Henri215 — APPROVED** [review]
Lewcs Good To Me

**Comment by Sadhorizon** [comment]
If I may suggest, remove them map-wise and add them to their locker's contents instead, like it is for other heads.

File: code/modules/pda/cart.dm [hunk 3]
```diff
@@ -158,15 +158,23 @@
...
+/obj/item/cartridge/qm
+	name = "Space Parts & Space Vendors Cartridge DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control and a status display controller."
+	icon_state = "cart-q"
+	programs = list(
+		new /datum/data/pda/app/supply,
+		new /datum/data/pda/app/mule_control,
+		new /datum/data/pda/app/status_display
+	)
+
```
Comment (Sirryan2002) [inline code/modules/pda/cart.dm:170]: should this be purchasable in the p-tech vendor?

Comment (Henri215) [reply code/modules/pda/cart.dm:170]: Probably not since this has access to status displays

File: code/modules/pda/cart.dm [hunk 4]
```diff
@@ -158,15 +158,23 @@
...
 /obj/item/cartridge/hop
 	name = "HumanResources9001"
-	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control, a custodial locator and a status display controller."
+	desc = "A data cartridge for portable microcomputers. Has security records, a custodial locator and a status display controller."
```
Comment (Contrabang) [inline code/modules/pda/cart.dm:173]: Oxford comma, my beloved
```suggestion
	desc = "A data cartridge for portable microcomputers. Has security records, a custodial locator, and a status display controller."
```

File: code/modules/pda/cart.dm [hunk 5]
```diff
@@ -158,15 +158,23 @@
 		new /datum/data/pda/app/status_display
 	)
 
+/obj/item/cartridge/qm
+	name = "Space Parts & Space Vendors Cartridge DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control and a status display controller."
```
Comment (Contrabang) [inline code/modules/pda/cart.dm:163]:
```suggestion
	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control, and a status display controller."
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Pull request merged**